testapps/integration/web/index.php

```diff
@@ -29,12 +29,12 @@
 });
 
 $app->post('/logging_standard', function (Request $request) {
-    $logName = $request->get('log_name');
-    $token = $request->get('token');
+    $logName = $request->request->get('log_name');
```

Comment:
Ok, it looks like this code can just be moved to the logging_custom test. Our logger doesn't look like it has a default log name"